CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing
+
+:partying_face::cowboy_hat_face::tada::+1: Welcome to MeetingMinder :+1::tada::cowboy_hat_face::partying_face:
+
+## Resources
+- [Getting Started / Setup](https://github.com/SocexSolutions/agenda-v2/blob/master/README.md)
+- [Docs](https://github.com/SocexSolutions/agenda-v2/tree/master/docs)
+- [Issues](https://github.com/SocexSolutions/agenda-v2/issues)
+
+## Summary
+
+When contributing to MeetingMinder, if there is not already an issue open, please first discuss the change you wish to make via issue before making a change. 
+
+Please note we have a [code of conduct](https://github.com/SocexSolutions/agenda-v2/blob/master/CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+## Testing
+
+### Frontend:
+
+- Spend some time testing what your changes affected. 
+- Try to break your own feature. 
+- Think about edge cases.
+
+### Backend:
+- make sure any additional features or changes to existing features are covered under the tests in `api/test/tests`
+- run `npm run test` in console and confirm all tests are passing
+- ensure changes function as expected on the front end
```

Comment:
Should these all be capitalized? Also, I think there should be a newline after `### Backend` for consistency.